.github/workflows/checkpatch.yml

```diff
@@ -0,0 +1,33 @@
+name: Check coding style
+on: [pull_request]
+jobs:
+  check-format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Try to fetch required commits
+        run: |
+          set -xe
+          echo "base_ref=${{ github.base_ref }}"
+          echo "github.event.pull_request.head.sha=${{ github.event.pull_request.head.sha }}"
+          # Based on https://stackoverflow.com/questions/54181901/fetching-only-the-range-of-commits-not-present-in-base-branch
+          for ((depth = 8; depth <= 65536; depth *= 2)); do
```

Comment:
Hm, is bash guaranteed?

Reply:
Maybe dash supports it? Anyway I've updated the action to use bash by default.